Nataly_Bondarenko/HW6/task3_count_letters.py

```diff
@@ -0,0 +1,15 @@
+# Please count how many times each letter appears in this file
+from collections import Counter
+
+with open('text.txt', 'r') as file:
+    data = file.read()
+
+rep_list = ['-', '.', ' ', "'", "[", ":", '"', ']', 'â€', '”', '3', '2', ';',
+            '(', ')', '\n', '7', '0', ',']
```

Comment:
what if the text will have some char that is not in this list?

Mostly this solution is okay, but it is better to remove all charm which does not a letter using regex

Reply:
fixed